voc/virtual_machine.go

```diff
@@ -30,5 +30,6 @@ type VirtualMachine struct {
 	BlockStorage      []ResourceID       `json:"blockStorage"`
 	BootLogging       *BootLogging       `json:"bootLogging"`
 	MalwareProtection *MalwareProtection `json:"malwareProtection"`
-	OSLogging         *OSLogging         `json:"oSLogging"`
+	OSLogging         *OSLogging         `json:"osLogging"`
```

Comment:
You changed the name in the json tag: Confirm that it is intended?

Reply:
Yes it is.